modules/installing-aws-load-balancer-operator-cli.adoc

```diff
@@ -0,0 +1,133 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator-cli_{context}"]
+= Installing the AWS Load Balancer Operator with the command line
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
+
+* Your cluster is configured with AWS as the platform type and cloud provider.
+
+* You are logged into the command line client `oc`.
```

Comment:
Should write oc like below. See: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift-cli

```suggestion
* You are logged into the OpenShift CLI (`oc`).
```

Reply:
ty